core/src/camera.rs

```diff
@@ -150,6 +149,12 @@ fn camera_controller(
         middle_point.y = size.y / 2.0;
     }
 
+    // Reset camera_pos to middle_point if it's NaN
+    let camera_pos = &mut camera_shake.center;
+    if camera_pos.is_nan() {
+        *camera_pos = middle_point.extend(0.0);
```

Comment:
This solves the strange camera  issue with a single player dying. It would set the camera_pos to have an `x` of NaN causing all other math operations to be NaN. Resetting it to middle point for this edge case

Reply:
New Camera:


https://user-images.githubusercontent.com/9278174/213054621-5cde980c-46a9-4718-888a-245feb79047a.mp4

Reply:
Ah, that makes sense. There must be a division by zero somewhere in the code when all the players are dead at the same time. I haven't played it one player since I update the camera controller so that's why I hadn't seen it before.

I'm glad it's not something that only happens on some computers because that would not be good!

Good job finding it. Do you mind opening a separate PR with just the camera fix?

Reply:
https://github.com/fishfolk/jumpy/pull/569